src/overlays/actors/ovl_Demo_Ec/z_demo_ec.c

```diff
@@ -768,9 +775,9 @@ void DemoEc_UpdateDancingZora(DemoEc* this, GlobalContext* globalCtx) {
 }
 
 void DemoEc_DrawDancingZora(DemoEc* this, GlobalContext* globalCtx) {
-    static void* eyeTextures[] = { 0x06003E40, 0x06004640, 0x06004E40 };
+    static u64* eyeTextures[] = { gZoraEyeOpenTex, gZoraEyeHalfTex, gZoraEyeClosedTex };
     s32 eyeTexIndex = this->eyeTexIndex;
-    void* eyeTexture = eyeTextures[eyeTexIndex];
+    u64* eyeTexture = eyeTextures[eyeTexIndex];
```

Comment:
No need to revert it, but `void*` is actually the preferred type for texture references. (Try to use `void*` going forward if possible tho, to avoid increasing the amount of fixing we have to do in the future)